frontend/src/__tests__/cypress/cypress/utils/testConfig.ts

```diff
@@ -3,7 +3,12 @@ import path from 'path';
 import { env } from 'process';
 import dotenv from 'dotenv';
 import YAML from 'yaml';
-import type { UserAuthConfig, TestConfig } from '~/__tests__/cypress/cypress/types';
+import type {
+  UserAuthConfig,
+  TestConfig,
+  AWSS3BucketDetails,
+  AWSS3Buckets,
+} from '~/__tests__/cypress/cypress/types';
 
 [
   `.env.cypress${env.CY_MOCK ? '.mock' : ''}.local`,
```

Comment:
Can you add Env variable and a default value for the Pipeline S3 Bucket, maybe something like:
```
`.env.cypress${env.CY_AWS_BUCKET ? '.S3.BUCKET' : 'BUCKET_2'}.local`,
```

Then replace all `BUCKET_2` to `env.CY_AWS_BUCKET`.

This way can `export CY_AWS_BUCKET=ANY_BUCKET_NAME`, while the default value would be BUCKET_2. @christianvogt what do you think ?

Reply:
I'm indifferent on how to manage these types of env vars.